docs/sources/datasources/azuremonitor.md

```diff
@@ -10,49 +10,38 @@ weight = 300
 
-1. In the name field, Grafana will automatically fill in a name for the data source - `Azure Monitor` or something like `Azure Monitor - 3`. If you are going to configure multiple data sources, then change the name to something more informative.
+2. In the name field, Grafana will automatically fill in a name for the data source - `Azure Monitor` or something like `Azure Monitor - 3`. If you are going to configure multiple data sources, then change the name to something more informative.
 
-1. If you are using Azure Monitor, you need 4 pieces of information from the Azure portal (see link above for detailed instructions):
+3. If you are using Azure Monitor, you need 4 pieces of information from the Azure portal (see link above for detailed instructions):
 
    - **Tenant Id** (Azure Active Directory -> Properties -> Directory ID)
    - **Client Id** (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
    - **Client Secret** (Azure Active Directory -> App Registrations -> Choose your app -> Keys)
    - **Default Subscription Id** (Subscriptions -> Choose subscription -> Overview -> Subscription ID)
 
-1. Paste these four items into the fields in the Azure Monitor API Details section:
+4. Paste these four items into the fields in the Azure Monitor API Details section:
    {{< figure src="/static/img/docs/v62/config_1_azure_monitor_details.png" class="docs-image--no-shadow" caption="Azure Monitor Configuration Details" >}}
 
    - The Subscription Id can be changed per query. Save the data source and refresh the page to see the list of subscriptions available for the specified Client Id.
 
-1. If you are also using the Azure Monitor Logs service, then you need to specify these two configuration values (or you can reuse the Client Id and Secret from the previous step).
+5. If you are also using the Azure Monitor Logs service, then you need to specify these two configuration values (or you can reuse the Client Id and Secret from the previous step).
 
    - Client Id (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
    - Client Secret (Azure Active Directory -> App Registrations -> Choose your app -> Keys -> Create a key -> Use client secret)
 
-1. If you are using Application Insights, then you need two pieces of information from the Azure Portal (see link above for detailed instructions):
-
-   - Application ID
-   - API Key
-
-1. Paste these two items into the appropriate fields in the Application Insights API Details section:
-   {{< figure src="/static/img/docs/v62/config_2_app_insights_api_details.png" class="docs-image--no-shadow" caption="Application Insights Configuration Details" >}}
-
-1. Test that the configuration details are correct by clicking on the "Save & Test" button:
+6. Test that the configuration details are correct by clicking on the "Save & Test" button:
```

Comment:
```suggestion
1. Test that the configuration details are correct by clicking on the "Save & Test" button:
```

Reply:
I think `6.` is correct - this is the 6th step in the series, no?

Reply:
All numbered steps should be "1.", they get converted.